packages/mui-base/src/index.js

```diff
@@ -1,125 +1,45 @@
 'use client';
 export * from './utils';
-
-export { default as Badge } from './Badge';
 export * from './Badge';
-
-export { default as Button } from './Button';
 export * from './Button';
-
-export { default as ClickAwayListener } from './ClickAwayListener';
-
-export { default as unstable_composeClasses } from './composeClasses';
```

Comment:
This export is missing as far as I can see. We should check the `unstable_` exports with the codemod. They are `unstable_` but people may use them.

Reply:
The codemod isn't responsible for this part, it only covers exports from `@mui/base`, not local.

Reply:
Ok, but we should add back the export, right?